adapter/ipListChecker/adapter.go

```diff
@@ -40,7 +40,7 @@ func (a *adapterState) Description() string {
 
 func (a *adapterState) Close() error { return nil }
 
-func (a *adapterState) ValidateConfig(cfg aspect.Config) (ce *aspect.ConfigErrors) {
+func (a *adapterState) ValidateConfig(cfg adapter.Config) (ce *adapter.ConfigErrors) {
```

Comment:
Even though adapter.ConfigErrors work well here, 
they look out of place with aspect managers which also impement `ValidateConfig`
perhaps. `mixer.ConfigErrors` would work best.